drivers/staging/Makefile

```diff
@@ -57,3 +57,4 @@ obj-$(CONFIG_ANDROID)		+= android/
 obj-$(CONFIG_PHONE)		+= telephony/
 obj-$(CONFIG_RAMSTER)		+= ramster/
 obj-$(CONFIG_USB_WPAN_HCD)	+= ozwpan/
+obj-$(CONFIG_PRIMA_WLAN)	+= prima/
```

Comment:
Make it like this.

\# Wlan drivers
obj-$(CONFIG_PRIMA_WLAN)	+= prima/
obj-$(CONFIG_PRONTO_WLAN)	+= prima/